npm-shrinkwrap.json

```diff
@@ -1858,6 +2232,820 @@
       "resolved": "http://registry.npmjs.org/chokidar/-/chokidar-1.6.1.tgz",
       "dev": true,
       "dependencies": {
+        "fsevents": {
```

Comment:
(I also forget `--no-optional` all the time...)

I've experimented with yarn on [a new project](https://github.com/josephfrazier/octopermalinker) that I forked from OctoLinker, and [it seems to work out ok](https://travis-ci.org/josephfrazier/octopermalinker/builds/205676836). Maybe I'll open a pull request, so we can talk about it there.